app/views/books/_filters.html.erb

```diff
@@ -1,10 +1,17 @@
 <div class="row store-top-bar">
-  <div class="col-md-5 store-order-field">
-    <%= form_tag '/', method: :get do %>
-      Order books by:
-      <%= select_tag "sort_field", options_for_select(@books.first.sortable_fields), allow_blank: true %>
-      <%= select_tag "sort_order", options_for_select([['Ascending', 'ASC'],['Descending', 'DESC']]) %>
-      <%= submit_tag "Submit", class: "btn btn-primary" %>
-    <% end %>
-  </div>
+  <div class="col-md-2"><%= link_to 'Home', root_path %></div>
+
+    <div class="col-md-10 store-order-field">
+      <%= form_tag '/', method: :get do %>
```

Comment:
This is perfectly fine, but next time you have to build a form, try [simple_form](https://github.com/plataformatec/simple_form). It'll make your life much better :)